bin/bump_version.nim

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env -S nim r --verbosity:0 --skipParentCfg:on
```

Comment:
Some implementations of `env` support the `-S` flag, which allows running this file with just `./bump_version.nim`.